### ISSUE TITLE
Question about the network uncertainty

### ISSUE THREAD

**Issue by saya34** [idekerlab/DrugCell#19]
Hi. The Paper said the model can pick a set of important subsystems for drug response by RLIPP score, and the RLIPP score calculation depends on VNN model parameters. But we know the Neural Network has strong uncertainty, I mean though we use the same training data and set the same hyperparameters ( the same hyperparameters such as the same epoch number,learning rate,initial strategy ), we can get different model which has totally different model parameters.  I am curious that if we tried training the network twice, can we get the same results or the same subsystem RLIPP score ranking between the two network? If not, which result should we pick to explain the drug response mechanism, or what strategy do you suggest?
